webapp/views/wpt-results.js

```diff
@@ -255,6 +255,9 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
     <template is="dom-if" if="[[shouldDisplayToggle(canViewInteropScores, pathIsATestFile)]]">
       <div class="channel-area">
         <paper-button id="toggleInterop" class\$="[[ interopButtonClass(view) ]]" on-click="clickInterop">Interop View</paper-button>
+        <template is="dom-if" if="[[showViewEqTest]]">
```

Comment:
If not using a flag, showViewEqTest can instead check the presence of view=test parameter